js/Wallop.js

```diff
@@ -74,11 +77,11 @@
 
   // Update prev/next disabled attribute
   WS.updateButtonStates = function () {
-    if (!this.buttonPrevious && !this.buttonNext) { return; }
+    if ((!this.buttonPrevious && !this.buttonNext) || this.options.carousel) { return; }
```

Comment:
I have also updated the `updateButtonStates` to return directly when the `carousel` option is enabled.